common/utils.js

```diff
@@ -204,6 +204,124 @@ export function isOutOfViewPort (element) {
   isOut.all = Object.values(isOut).every(val => val);
   return isOut;
 }
+
```

Comment:
This file has just the regexes, which are equivalent to Dialpad backend, and few of the utils that aren't directly used by this feature but were added by Brad's suggestion